scripts/test.sh

```diff
@@ -42,7 +42,9 @@ ls -la build/
 # Run Ratel and Dgraph
 pushd "$composedir" > /dev/null
    set -e
-   docker-compose up --force-recreate --remove-orphans --detach
+   docker-compose down
+   sleep 6
+   docker-compose up --force-recreate --remove-orphans
```

Comment:
I think you need `--detach` argument. That would explain why teamcity builds get stuck -- without detach `compose up` blocks indefinitely.

Reply:
Yeah, I did that just for a test case. I will always keep the good practices there as soon I finish the work and troubleshooting.

I had to remove it due a bug with Docker. Check the commit message. I'll revert it soon.